src/cli/cli_dataset.cpp

```diff
@@ -118,6 +118,21 @@ otError Dataset::Print(otOperationalDataset &aDataset)
     return OT_ERROR_NONE;
 }
 
+/**
+ * @cli dataset init (active,new,pending,tlvs)
+ * @code
+ * dataset init new
+ * Done
+ * @endcode
+ * @cparam dataset init {@ca{active}|@ca{new}|@ca{pending}|@ca{tlvs}} [@ca{hex-encoded-tlvs}]
+ * Use `new` to initialize a new dataset, then enter the command `dataset commit active`.
+ * Use `tlvs` for hex-encoded TLVs.
+ * @par
+ * OT CLI checks for `active`, `pending`, or `tlvs` and returns the corresponding values. Otherwise,
+ * OT CLI creates a new, random network and returns a new dataset. Can be used with OTBR.
```

Comment:
Most of these commands can be used on OTBR. What are we trying to convey with "Can be used with OTBR."?

Reply:
I removed this...